### PR TITLE
created catalog

### PR DESCRIPTION
**Pull request by yevheniia-biliaieva**
- [DEMO LINK](https://yevheniia-biliaieva.github.io/layout_catalog/)
- [TEST REPORT LINK](https://yevheniia-biliaieva.github.io/layout_catalog/report/html_report/)

- [x] All component follow BEM and use SCSS
- [x] repaeted sizes and special colors are put to variables
- [x] Grid is used for the columns
- [x] cards are shown in 1, 2, 3 or 4 columns based on screen resolution
- [x] All changes on `:hover` are smooth
- [x] Code follows all the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)

### CODE REVIEW

File: src/styles/blocks/header.scss [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    color: $text-main-color;
+    text-transform: uppercase;
+    text-decoration: none;
+  }
+
+  &__link:hover {
+    @include smooth-changes;
+
+    color: $accent-color;
+  }
```
Comment (lerastarynets) [inline src/styles/blocks/header.scss:44]: use nesting
```suggestion
    text-decoration: none;
      &:hover {
        @include smooth-changes;
    
        color: $accent-color;
      }
    }
```

File: src/styles/blocks/header.scss [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+  &__item {
+    margin-left: 20px;
+    position: relative;
+  }
+
+  &__item:first-child {
+    margin: 0;
+  }
```
Comment (lerastarynets) [inline src/styles/blocks/header.scss:29]: make use of nesting
```suggestion
  &__item {
    margin-left: 20px;
    position: relative;
        &:first-child {
          margin: 0;
       }
  }
```
**PLS DO THE SAME FOR ALL SUCH CASES IN THE PROJECT**

**Review by lerastarynets — CHANGES_REQUESTED** [review]
Nice start! Let's fix and improve your work a little bit)

File: src/styles/blocks/header.scss [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  &__link {
+    font-size: 12px;
+    display: inline-block;
+    padding: 23px 0;
+    color: $text-main-color;
+    text-transform: uppercase;
+    text-decoration: none;
+
+    &:hover {
+      @include smooth-changes;
```
Comment (IvanFesenko) [inline src/styles/blocks/header.scss:40]: Transituon should work on both ways, not only on hover
```suggestion
    text-decoration: none;
    @include smooth-changes;
    
    &:hover {

```

File: src/styles/blocks/main.scss [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+  }
+
+  @include on-laptop {
+    grid-template-columns: repeat(4, 1fr);
+  }
+}
+
+.product-card {
+  width: 200px;
+  box-sizing: border-box;
```
Comment (IvanFesenko) [inline src/styles/blocks/main.scss:28]: Move   `box-sizing: border-box;` into `*` selector

File: src/styles/blocks/main.scss [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+
+  &__img {
+    align-self: center;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__product-name {
+    font-weight: 450;
```
Comment (IvanFesenko) [inline src/styles/blocks/main.scss:50]: wrong value - it should be 400 or 500

File: src/styles/blocks/main.scss [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+    line-height: 16px;
+    color: #fff;
+    text-align: center;
+
+    padding: 12px 70px;
+    border-radius: 5px;
+    background: $accent-color;
+
+    &:hover {
+      @include smooth-changes;
```
Comment (IvanFesenko) [inline src/styles/blocks/main.scss:111]: Same here

File: src/styles/blocks/main.scss [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+
+      border: 1px solid $accent-color;
+      color: $accent-color;
+      background: $bg-color;
+    }
+  }
+
+  &:hover {
+    @include smooth-changes;
```
Comment (IvanFesenko) [inline src/styles/blocks/main.scss:120]: and here

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
Something wrong with styles on demo link
2023-12-18_16h09_13

File: src/styles/blocks/main.scss [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+  @include on-tablet {
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  @include on-laptop {
+    grid-template-columns: repeat(4, 1fr);
+  }
+}
+
+.product-card {
```
Comment (etojeDenys) [inline src/styles/blocks/main.scss:26]: better to move it to separate file

**Review by etojeDenys — APPROVED** [review]
good job

**Pull request closed without merging**